include/openenclave/bits/sgx/region.h

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _OE_INTERNAL_REGION_H
+#define _OE_INTERNAL_REGION_H
+
+#include <openenclave/bits/defs.h>
+#include <openenclave/bits/result.h>
+#include <openenclave/bits/sgx/sgxtypes.h>
+#include <openenclave/bits/types.h>
+
+OE_EXTERNC_BEGIN
+
+#define OE_MAX_REGIONS 16
+
+typedef struct oe_region_context oe_region_context_t;
```

Comment:
Any SGX-only defines should have sgx in the name.  But I wonder if some of the things in this file aren't really SGX only and should be moved outside the sgx subdir.